ReactNativeClient/lib/markdownUtils.js

```diff
@@ -91,6 +91,15 @@ const markdownUtils = {
 
 		return output.join('\n');
 	},
+
+	filterLink(mdLink) {
```

Comment:
Why call if "filterLink" when it doesn't even filter links? Maybe filterImage, but something more specific like "titleFromBody" would be better.

Reply:
And why is the input called "mdLink". Does your function not work if the input is not a link?

Reply:
Now it returns the cleaned title from the body